enterprise/server/remote_execution/platform/platform.go

```diff
@@ -81,6 +81,7 @@ const (
 	unsetContainerImageVal = "none"
 
 	RecycleRunnerPropertyName               = "recycle-runner"
+	RecycleRunnerKeyPropertyName            = "recycle-runner-key"
```

Comment:
We've recommended `runner-recycling-key` in the past to some customers and it's also what we're currently using in our repo. Can you update this const + the docs?

```suggestion
	RunnerRecyclingKey            = "runner-recycling-key"
```